in_toto/runlib.go

```diff
@@ -27,7 +28,9 @@ using sha256 and returns a map in the following format:
 	}
 
 If reading the file fails, the first return value is nil and the second return
-value is the error.
+value is the error. For multi operating system support, we normalize the
+line separators of all files. This means, that Windows CRLF line separators
+will being transformed to Linux LF line separators.
```

Comment:
What do you think about ...?
```suggestion
value is the error.
NOTE: For cross-platform consistency Windows-style line separators (CRLF) are
normalized to Unix-style line separators (LF) before hashing file contents.
```

Reply:
sounds good. I will make the necessary changes.